pkg/downloader/manager.go

```diff
@@ -206,6 +206,7 @@ func (m *Manager) downloadAll(deps []*chartutil.Dependency) error {
 	}
 
 	destPath := filepath.Join(m.ChartPath, "charts")
+	tmpPath := filepath.Join(m.ChartPath, "tmpcharts")
```

Comment:
If we want to make it safe for concurrent calls we should think about introducing some kind of lock file which would make commands tamper with chart dir sequentially.